### PR TITLE
data: RockingSNP

### PR DESCRIPTION
**Pull request by RockingSNP**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4639"></a>

### CODE REVIEW

**Comment by RockingSNP** [comment]
Cannot pass the last check

File: data/RockingSNP.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    },
+    {
+      "name": "Follow me on Github",
+      "url": "https://www.github.com/RockingSNP/",
+      "icon": "Fagithub"
+    },
+  {
+      "name": "Connect with me on LinkedIn",
+      "url": "https://www.linkedin.com/in/souvik-p-117647218/",
+      "icon": "FaLinkedIn"
```
Comment (Sindhuinti) [inline data/RockingSNP.json:21]: also change the `FaLinkedIn` to `Falinkedin`

**Review by Sindhuinti — CHANGES_REQUESTED** [review]

File: data/RockingSNP.json [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+"links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/SouvikP74860962",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Follow me on Github",
+      "url": "https://www.github.com/RockingSNP/",
+      "icon": "Fagithub"
```
Comment (Krishnapro) [inline data/RockingSNP.json:16]:
```diff
-"icon": "Fagithub"
+"icon": "FaGithub"
```

please fix the typo in Github icon

**Review by Krishnapro — CHANGES_REQUESTED** [review]

File: data/RockingSNP.json [hunk 3]
```diff
@@ -0,0 +1,53 @@
+{
+  "name": "Souvik Pramanick",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Full Stack Web Developer **Digital Nomad**. Explporer & Karma -te Prastut"
```
[the remark this answers is not in the document]
Comment (RockingSNP) [reply data/RockingSNP.json:5]: Thnx

**Comment by RockingSNP** [comment]
`FaLinkedIn` changed to `FalikedIn`
`Fagithub` changed to `FaGithub`
Comma added

File: data/RockingSNP.json [hunk 4]
```diff
@@ -0,0 +1,53 @@
+{
+  "name": "Souvik Pramanick",
+  "displayStatsPublic": true,
```
Comment (kumarsonsoff3) [inline data/RockingSNP.json:3]: As of our latest update, we no longer need the `displayStatsPublic` key. So please remove this field from your json. 

```suggestion
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

File: data/RockingSNP.json [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+ {
+      "name": "Subscribe me on Youtube",
+      "url": "https://www.youtube.com/@souvikpramanick1549/",
+      "icon": "FaYoutube"
+    },
+{
+      "name": "Follow me on Instagram",
+      "url": "https://www.instagram.com/rockingsp_official/",
+      "icon": "FaInstagram"
+    },
```
Comment (kumarsonsoff3) [inline data/RockingSNP.json:47]: Also remove this extra comma `,` from here. 

```suggestion
    }
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

**Comment by RockingSNP** [comment]
Is it good now?

**Comment by RockingSNP** [comment]
Merge this please

File: data/RockingSNP.json [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+    {
+      "name": "Follow me on Github",
+      "url": "https://www.github.com/RockingSNP/",
+      "icon": "FaGithub"
+    },
+  {
+      "name": "Connect with me on LinkedIn",
+      "url": "https://www.linkedin.com/in/souvik-p-117647218/",
+      "icon": "FalinkedIn"
```
Comment (kumarsonsoff3) [inline data/RockingSNP.json:20]: Looks good to me. 

>**Note**: You're using an `invalid` Linkedin icon here. It should be `FaLinkedin`, else the fallback icon FaGlobe will be used by default on your page. 
```suggestion
      "icon": "FaLinkedin"
```

**Review by kumarsonsoff3 — DISMISSED** [review]

**Review by kumarsonsoff3 — APPROVED** [review]

**Pull request merged**